users/models.py

```diff
@@ -9,7 +9,7 @@
 class UserManager(BaseUserManager):
 
     def create_user(self, email, first_name,
-                    last_name, birth_date, password,
+                    last_name, birth_date, password, user_type,
```

Comment:
I think that user-type will be sent into create_user with other_fields anyway.
I'm not sure that we need to send it explicitly - because we will create a user with a form anyway, so I think sending only the essentials is better. (fields that don't have a default options)

Reply:
@Micha546 I'm not sure i'm getting it. If I take this down then wont 'user_type' be neglected from the form?
I don't think any user will want to be set as default for seeker during his sign up.

Reply:
I think that if blank=False - which in our case is true because it's the default option, then in a form the user will have to fill owner or seeker in the user_type options.

The create_user function is called after the form and the user_type will pass through in the "other_fields" dictionary - with either owner or seeker as its value

I think this is how it works, I'm not 100% sure, maybe a red-hatter can confirm?

Reply:
After a small run and check, it's no problem to manually add any fields we like to the form,
and the 'user_type' field seems to act great in such case!
I will remove the variable and it shall pass through 'other fields'.
@Micha546 please review all my small changes as well and mark this as resolved if it's to your liking and fits your tests.

Reply:
Everything looks 🔥